src/monopoly/cards/chanceDeck.js

```diff
@@ -13,24 +13,28 @@ inherits(ChanceDeck, TableTop.Deck);
 var buildChanceDeck = function() {
 
   var chance1 = new TableTop.Card("Advance to Go (Collect $200)", function(game) {
-    game.getCurrentPlayer().moveTo(0);
+    var player = game.getCurrentPlayer();
+    game.moveTo(0, player);
     return game.board.tiles[0].performLandingAction(game);
   });
 
   var chance2 = new TableTop.Card("Advance to North Mass.", function(game) {
-    game.getCurrentPlayer().moveTo(24);
+    var player = game.getCurrentPlayer();
+    game.moveTo(24, player);
     return game.board.tiles[24].performLandingAction(game);
   });
 
   var chance3 = new TableTop.Card("Advance token to the nearest Utility. If unowned, you may buy it from the Bank. If owned, throw dice and pay owner a total of ten times the amount thrown.", function(game) {
```

Comment:
Maybe you should test it out before you go saying that you won't collect $200 lol. I think that should work. Automatic via the moveTo method.